### DISCUSSION TITLE
Hello, is this power friendly to laptops comparing it to Edge's capabilities?

### DISCUSSION THREAD

**Discussion by AlexDeFoc** [Alex313031/thorium#309] [Q&A] [answered]
Hello, I'm Alex, and I want ask if this browser if used on a laptop would use less or more battery then edge, chrome (its worse then edge)

I concluded that Edge is the best at power saving, but I am very interested in using this browser as my main and would like to use it on my laptop too.

**Comment by midzer** [comment, marked as answer]
This is kinda hard to measure in a "real life environment".

Thorium's performance benefit (to other Chrom* based browsers) depends very much on the specific OS and CPU architecture, especially regarding JavaScript execution speed.

There are regular benchmarks on https://thorium.rocks/performance probably a lil outdated though.
